src/controllers/author.controller.ts

```diff
@@ -1,9 +1,12 @@
 import { Request, Response, NextFunction } from 'express';
 import asyncHandler from 'express-async-handler';
 
+import * as services from '../services/author.services';
```

Comment:
define alias rõ ràng em nhé
```suggestion
import * as authorServices from '../services/author.services';
```